webapp/store/views.py

```diff
@@ -1,27 +1,43 @@
-from pprint import pformat
+from flask import Blueprint, render_template
+from webapp.store import models
 
-from flask import Blueprint, abort, render_template
-
-from theblues.charmstore import CharmStore
 
 jaasstore = Blueprint(
   'jaasstore', __name__,
   template_folder='/templates', static_folder='/static')
 
-cs = CharmStore("https://api.jujucharms.com/v5")
-
 
 @jaasstore.route('/store')
 def store():
     return render_template('store/store.html')
 
 
-@jaasstore.route('/<entity_name>')
-def details(entity_name):
-    try:
-        entity = cs.entity(entity_name)
-        entity_formatted = pformat(entity, 1, 120)
-    except Exception:
-        return abort(404, "Entity not found {}".format(entity_name))
+@jaasstore.route('/u/<user_name>/<entity_name>/<series>/<version>')
```

Comment:
This should be in its own blueprint

Reply:
@tbille didn't you see the `NotImplementedError`? I think it's reasonable to not do further work on this when it's explicitly not implemented yet.

Reply:
Ok but let's keep it